support/jsdoc/theme/tmpl/layout.tmpl

```diff
@@ -85,8 +78,18 @@
     Documentation generated by <a href="https://github.com/jsdoc3/jsdoc">JSDoc <?js= env.version.number ?></a><?js if(env.conf.templates && env.conf.templates.default && env.conf.templates.default.includeDate !== false) { ?> on <?js= (new Date()) ?><?js } ?> using the Minami theme.
 </footer>
 
+
+<script src="scripts/prettify/prettify.js"></script>
+<script src="scripts/prettify/lang-css.js"></script>
```

Comment:
@hargasinski do you know if these scripts have the `async` attribute or do they depend on one another?